view-creator-framework/build.gradle.kts

```diff
@@ -85,4 +76,9 @@ dependencies {
   testImplementation("org.mockito:mockito-core:4.5.1")
 }
 
+// Disabling compatibility check for the test avro definitions.
+tasks.named<org.hypertrace.gradle.avro.CheckAvroCompatibility>("avroCompatibilityCheck") {
```

Comment:
Is this apply only to test?

Reply:
we have only test avdl generation here in this module.